src/collective/taxonomy/utility.py

```diff
@@ -302,3 +310,16 @@ def translate(self, msgid, mapping=None, context=None,
             pretty_path = pretty_path.rsplit(PRETTY_PATH_SEPARATOR, 1)[-1]
 
         return pretty_path
+
+    def _fixup(self):
+        # due to compatibility reasons this method fixes data structure
+        # for old Taxonomy instances.
+        # XXX: remove this in version 2.0 to prevent write on read
+        if self.order is None:
+            safeWrite(self, getRequest())
```

Comment:
Great. Poor old `self.REQUEST` – acquisition has fallen out of favor...

Reply:
strange .. I got an `AttributeError` when doing `self.REQUEST` in _fixup ?

Reply:
I guess the utility is not acquisition-wrapped. Makes sense I guess.

Plone is really split brain in terms of acquisition and components.